packages/abi-gen-wrappers/src/generated-wrappers/asset_proxy_owner.ts

```diff
@@ -1292,10 +1343,12 @@ export class AssetProxyOwnerContract extends BaseContract {
             // tslint:enable boolean-naming
             return result;
         },
-
         /**
-         * Returns the ABI encoded transaction data needed to send an Ethereum tx calling this method. Before sending the Ethereum tx, this encoded tx data can first be sent to a separate signing service or can be used to create a 0x transaction (see protocol spec for more details).
-         * @param _secondsTimeLocked         Duration needed after a transaction is confirmed and before it becomes executable, in seconds.
+         * Returns the ABI encoded transaction data needed to send an Ethereum transaction calling this method. Before
+         * sending the Ethereum tx, this encoded tx data can first be sent to a separate signing service or can be used
+         * to create a 0x transaction (see protocol spec for more details).
+         * @param _secondsTimeLocked Duration needed after a transaction is confirmed
+         * and before it becomes executable, in seconds.
```

Comment:
My aesthetic eye would like to see wrapped lines (here "and before...") indented 4 spaces, aka a hanging indent.  But if this is convention, and, more importantly, if the generated docs look fine, then whatever.